### PR TITLE
[incubator-kie-issues-1131] test migration from V7 to code generation-5

### PR DESCRIPTION
**Pull request by Abhitocode**
In addition to previous changes associated,
migrated some BPMN processes which are being compiled using legacy v7 to code generation approach, using new branch incubator-kie-issues-1131-5

The tests can be identified by referring to Activitytest.java:
https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/ActivityTest.java

Closes https://github.com/apache/incubator-kie-issues/issues/1131.

### CODE REVIEW

**Comment by elguardian** [comment]
@Abhitocode plz fix the conflict

File: jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/ActivityTest.java [hunk 1]
```diff
@@ -976,48 +991,55 @@ public void testCallActivityWithBoundaryEvent() throws Exception {
...
+        assertThat(workItem).isNotNull();
+        subProcessInstance.completeWorkItem(workItem.getStringId(), Collections.emptyMap());
+        assertThat(processInstance).extracting(ProcessInstance::status).isEqualTo(ProcessInstance.STATE_COMPLETED);
+        Collection<String> processNodes = process.findNodes(Objects::nonNull).stream().map(Node::getName).collect(Collectors.toSet());
+        Collection<String> subProcessNodes = callActivitySubProcessWithBoundaryEventProcess.findNodes(Objects::nonNull).stream().map(Node::getName).collect(Collectors.toSet());
+        System.out.println("process nodes: " + processNodes);
+        System.out.println("sub process nodes:" + subProcessNodes);
+        //the following 3 are expected in actual response, but end event is not there, if we remove end process its passing or else not passing
+        //Actual response- EndProcess should be in the actual response.
+        assertThat(processNodes.containsAll(List.of("StartProcess", "Call Activity 1", "EndProcess"))).isTrue();
```
Comment (elguardian) [inline jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/ActivityTest.java:1018]: this is not proper translation. EventTracker Listener and the user
assertThat(listener.tracked()).anyMatch(ProcessTestHelper.triggered(<name of the node>)...

**Comment by kie-ci3** [comment]
**PR job** `#5` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-runtimes -u https://github.com/apache/incubator-kie-kogito-runtimes/pull/3559 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-runtimes-pr/job/PR-3559/5/display/redirect

**Test results:**
- PASSED: 2853
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-runtimes-pr/job/PR-3559/5/testReport/io.quarkus.it.kogito.devmode/DevMojoIT/testDRLHotReload/">io.quarkus.it.kogito.devmode.DevMojoIT.testDRLHotReload</a></summary>
Condition with io.quarkus.it.kogito.devmode.DevMojoIT was not fulfilled within 2 minutes.
</details>

**Review by elguardian — APPROVED** [review]
LGTM

**Comment by elguardian** [comment]
@martinweiler or @pefernan do you mind to have a look

**Review by pefernan — APPROVED** [review]

**Pull request merged**